cloud_guardian/iam_model/graph/identities/user.py

```diff
@@ -40,6 +52,12 @@ def from_dict(cls, user_dict: Dict[str, str]) -> User:
             create_date=datetime.fromisoformat(user_dict["CreateDate"]),
         )
 
+    def get_user(self, user_name: str) -> Optional[User]:
```

Comment:
why can't u use "get_or_create"? It will retrieve the instance or create a new one, do we need a get_user?

Reply:
I am currently refactoring it accordingly.

Reply:
Done.